tests/backup_restore_test.go

```diff
@@ -232,7 +221,7 @@ func TestPITR(t *testing.T) {
 		// 5. check that query from the database that had dropped will fail
 		rows, err := db.Query(fmt.Sprintf(`SHOW DATABASES LIKE '%s';`, database))
 		a.NoError(err)
-		defer rows.Close()
+		defer func() { a.NoError(rows.Close()) }()
```

Comment:
Instead, ignore them in https://github.com/bytebase/bytebase/blob/main/.golangci.yaml#L21

Reply:
Removed.
Actually they are not required by golangci-lint :joy: